crates/swc_ecma_minifier/tests/fixture/issues/firebase-core/1/output.js

```diff
@@ -1314,7 +1314,7 @@
                 let b1;
                 d || (c = (b1 = "", xa(c, function(c, d) {
                     b1 += d, b1 += ":", b1 += c, b1 += "\r\n";
```

Comment:
`b1` can have different value after evaluating

```js
                xa(c, function(c, d) {
                    b1 += d, b1 += ":", b1 += c, b1 += "\r\n";
                })
```

Reply:
Oh, this is indeed wrong